contracts/test/tx_allow_list.ts

```diff
@@ -68,4 +68,18 @@ describe("ExampleTxAllowList", function () {
   test("should not let allowed to revoke admin", "step_onlyAdminCanRevoke")
 
   test("should let admin to revoke allowed", "step_adminCanRevoke")
+
+  test("should let manager to revoke allowed", "step_managerCanRevoke")
```

Comment:
Can we put all of the tests in the same order here that they are declared on the `.sol` contract file?

Reply:
done (at least for these tests)